MobiFlight/BoardDefinitions.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
```

Comment:
This is a simple class that loads the board definitions from disk and then provides methods for looking up matching board definitions by different properties.